src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificateAssetDownloader.cs

```diff
@@ -130,55 +130,118 @@ internal static class CertificateAssetDownloader
 
                 // Get the relevant types needed.
                 Type? socketsHttpHandlerType = Type.GetType("System.Net.Http.SocketsHttpHandler, System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
+                Type? httpMessageHandlerType = Type.GetType("System.Net.Http.HttpMessageHandler, System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
```

Comment:
This method is getting pretty complicated with all this reflection.  As a follow-up, I wonder if it'd be worthwhile adding a single internal method to SocketsHttpHandler that does all this logic, and then just reflect to invoke that one member from here.

Reply:
Apparently part of me still believes this is Core 1.0, because my initial thought was "but what about package shear?".  Since we're both inbox I guess that's safe enough :smile: